### PR TITLE
Test function for Ekaterina (model refactor)

### PR DESCRIPTION
**Pull request by lyriccoder**
Test function which takes list of filenames and returns list of predictions

@KatGarmash, See tests, I have provided an example of usage. I will delete tests later

->  `model.test(files)`

You will return

```
[
		"D:\\git\\aibolit\\test\\model\\start_end\\UpDirective.java",
		[
			"P13",
			"P1",
			"P2",
			"P3",
			"P4",
			"P5",
			"P6",
			"P7",
			"P8",
			"P9",
			"P10",
			"P11",
			"P12",
			"P14",
			"P15",
			"P16",
			"P17",
			"P18",
			"P19",
			"P20_5",
			"P20_7",
			"P20_11",
			"P21",
			"P22",
			"P23",
			"P24",
			"P25",
			"P26",
			"P28",
			"P29",
			"P30",
			"P31",
			"P32"
		],
		[
			5.8,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0,
			0
		]
	], ...
```

### CODE REVIEW

File: aibolit/model/model.py [hunk 1]
```diff
@@ -80,20 +80,34 @@ def __init__(self):
         self.model = None
         self.features_conf = None
 
-    def fit_regressor(self, X, y, display=False):
+    def predict(self, input_params):
```
Comment (acheshkov) [inline aibolit/model/model.py:83]: what is the type of input_params?

File: aibolit/model/model.py [hunk 2]
```diff
@@ -159,7 +173,46 @@ def calculate_score(self, X, quantity_func='log', th=1.0, feature_importances=No
 
         return (np.array(ranked), pairs[:, 0].T.tolist()[::-1])
 
-    def rank(self, snippet, scale=True, th=1.0):
+    def test(self, files: List[str]):
```
Comment (acheshkov) [inline aibolit/model/model.py:176]: what does this function do?

File: aibolit/model/model.py (hunk 1 above)
Comment (acheshkov) [reply aibolit/model/model.py:83]: add type for returning value

File: aibolit/model/model.py (hunk 2 above)
Comment (acheshkov) [reply aibolit/model/model.py:176]: add type for returning value

Comment (lyriccoder) [reply aibolit/model/model.py]: fixed

File: aibolit/model/model.py (hunk 1 above)
Comment (lyriccoder) [reply aibolit/model/model.py]: fixed

**Comment by acheshkov** [comment]
@rultor merge

**Pull request merged**